src/components/uicomposition/TilesContainer.astro

```diff
@@ -37,9 +37,13 @@ const {
     )
   }
   <div class="flex w-full flex-col items-start justify-start">
-    {headline && !cta && <h2 class="font-serif text-2xl">{headline}</h2>}
     {
-      headline && cta && (
+      headline && headline.length > 0 && (
```

Comment:
something is not right here, as it sometimes renders the headline twice now: 
![image](https://github.com/user-attachments/assets/cf014158-f695-4fd2-a89b-94f7a9de0d66)

Reply:
Fixed it